source/apps.jsonld

```diff
@@ -97,7 +97,8 @@
         { "alias": "includeEplikt", "filter": "NOT excludeEplikt", "prefLabelByLang": { "sv": "Inkludera elektroniska pliktleveranser", "en": "Include electronic legal deposit"  } },
         { "alias": "excludePreliminary", "filter": "NOT encodingLevel:(\"marc:PartialPreliminaryLevel\" OR \"marc:PrepublicationLevel\")", "prefLabelByLang": { "sv": "Exkludera kommande publiceringar", "en": "Exclude upcoming publications" } },
         { "alias": "includePreliminary", "filter": "NOT excludePreliminary", "prefLabelByLang": { "sv": "Inkludera kommande publiceringar", "en": "Include upcoming publications" } },
-        { "alias": "existsImage", "filter": "image:*", "prefLabelByLang": { "sv": "Endast resurser med omslags-/miniatyrbild", "en": "Resources with cover/thumbnail only" } }
+        { "alias": "existsImage", "filter": "image:*", "prefLabelByLang": { "sv": "Endast resurser med omslags-/miniatyrbild", "en": "Resources with cover/thumbnail only" } },
+        { "alias": "freeDigital", "filter": "hasInstanceType:\"Electronic\" AND (usageAndAccessPolicy.label:\"gratis\" OR \"associatedMedia.marc:publicNote\":\"*fritt tillgänglig*\")", "prefLabelByLang": { "sv": "Fritt digitalt", "en": "Free digital material" } }
```

Comment:
`"fritt tillgänglig\"` (without the asterisks) should be sufficient?

Reply:
Also no need for quotation marks around `Electronic` and `gratis`!